# Pass the right flags to apr-config and apu-config.
# Only Solaris seems to need this, so not for upstream.
--- configure	2015-12-07 23:53:20.000000000 -0800
+++ configure	2016-03-20 15:21:11.528922510 -0700
@@ -4896,14 +4896,14 @@
   fi
 
   if test "$enable_all_static" = "yes"; then
-    SVN_APR_LIBS="`$apr_config --link-ld --libs`"
+    SVN_APR_LIBS="`$apr_config --link-libtool --libs`"
     if test $? -ne 0; then
-      as_fn_error $? "apr-config --link-ld --libs failed" "$LINENO" 5
+      as_fn_error $? "apr-config --link-libtool --libs failed" "$LINENO" 5
     fi
   else
-    SVN_APR_LIBS="`$apr_config --link-ld`"
+    SVN_APR_LIBS="`$apr_config --link-libtool`"
     if test $? -ne 0; then
-      as_fn_error $? "apr-config --link-ld failed" "$LINENO" 5
+      as_fn_error $? "apr-config --link-libtool failed" "$LINENO" 5
     fi
   fi
   SVN_APR_LIBS="`
@@ -5140,14 +5140,14 @@
   fi
 
   if test "$enable_all_static" = "yes"; then
-    SVN_APRUTIL_LIBS="`$apu_config --link-ld --libs`"
+    SVN_APRUTIL_LIBS="`$apu_config --link-libtool --libs`"
     if test $? -ne 0; then
-      as_fn_error $? "apu-config --link-ld --libs failed" "$LINENO" 5
+      as_fn_error $? "apu-config --link-libtool --libs failed" "$LINENO" 5
     fi
   else
-    SVN_APRUTIL_LIBS="`$apu_config --link-ld`"
+    SVN_APRUTIL_LIBS="`$apu_config --link-libtool`"
     if test $? -ne 0; then
-      as_fn_error $? "apu-config --link-ld failed" "$LINENO" 5
+      as_fn_error $? "apu-config --link-libtool failed" "$LINENO" 5
     fi
   fi
   SVN_APRUTIL_LIBS="`
